app/lib/actions/katello/organization/abstract_manifest_task.rb

```diff
@@ -0,0 +1,35 @@
+module Actions
+  module Katello
+    module Organization
+      class AbstractManifestTask < Actions::AbstractAsyncTask
```

Comment:
Using class hierarchy for this doesn't seem quite right.  This is 'adding' a feature, and i think a mix-in would be better.  Thoughts/